content/docs/api-reference/get_params.md

```diff
@@ -0,0 +1,90 @@
+# dvc.api.get_params()
+
+Returns all the <abbr>params</abbr> associated with the given
+<abbr>stage</abbr>.
+
+```py
+def get_params(stage: str, flatten: bool = False) -> Dict:
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.get_params("train")
```

Comment:
how will it behave if params come from Python? btw, I assume that it reads the workspace (rather than dvc.lock), is it correct?

Reply:
> how will it behave if params come from Python?

Only values that DVC can parse will be returned. Complex expressions (https://dvc.org/doc/command-reference/params#examples-python-parameters-file) won't be parsed.

> btw, I assume that it reads the workspace (rather than dvc.lock), is it correct?

Correct.

Reply:
Should it also be possible to read params from a particular revision and/or repo? It seems consistent with the rest of the API.

Reply:
Personally, I was interested to see if we can cover both the CLI-like use cases and the repro scenario to keep the API small and make public the CLI-like features that aren't documented. If it contradicts your use case, it makes sense to leave it out. In the case of making the stage optional, or including optional kwargs for repo or rev, would it break any existing functionality? Does it add a lot more complexity? If so, it's fine to leave as is.

Reply:
The use case I had in mind is, in conjunction with a `get_metrics()`, method, this would enable users to do custom analysis of experiments without having to manually clone the repo and use the CLI.

Reply:
> Personally, I was interested to see if we can cover both the CLI-like use cases and the repro scenario to keep the API small and make public the CLI-like features that aren't documented.

Makes sense.

> would it break any existing functionality? Does it add a lot more complexity? 

I don't think so but I will implement and see.

> The use case I had in mind is, in conjunction with a get_metrics(), method, this would enable users to do custom analysis of experiments without having to manually clone the repo and use the CLI.

Got it

Reply:
> maybe we should merge and add these in a follow-up

Referring to https://github.com/iterative/dvc/pull/7613 right? Not sure we want to document a WIP feature or not in great detail.